constants.js

```diff
@@ -1,9 +1,28 @@
-const pending = '-';
+const pending = '🛑';
 const passing = '✓';
-const failed = '❌';
+const failing = '❌';
+const accessDenied = '⊘';
+const timeout = '⏱';
+const notImplemented = '—';
+
+const statusMarks = {
+  pending,
+  passing,
+  failing,
+  accessDenied,
+  timeout,
+  notImplemented,
+  // this mocha's test states aren't in present tense
+  'passed': passing,
```

Comment:
```suggestion
  passed: passing,
```

Reply:
should be addressed here: https://github.com/digitalbazaar/mocha-w3c-interop-reporter/pull/7/commits/4585e53dd271bcf777588d558747dd6eb3366102